### PR TITLE
UseDungeonTextureTable for RuntimeMaterials

### PR DESCRIPTION
**Pull request by TheLacus**
Added option for RuntimeMaterials to use the texture table of parent dungeon.

### CODE REVIEW

File: Assets/Scripts/DungeonTextureTables.cs [hunk 1]
```diff
@@ -89,5 +89,35 @@ public static int[] RandomTextureTableAlternate(int seed)
 
             return textureTable;
         }
+
+        /// <summary>
+        /// Applies the requested dungeon texture table to texture arhive index.
```
Comment (XJDHDR) [inline Assets/Scripts/DungeonTextureTables.cs:94]: Small typo here: "arhive" should be "archive"

**Comment by XJDHDR** [comment]
As I reported on the forum, there may be a problem with this PR as it currently stands: [https://forums.dfworkshop.net/viewtopic.php?p=46217#p46217](https://forums.dfworkshop.net/viewtopic.php?p=46217#p46217)

**Comment by Interkarma** [comment]
Just checking on status of this one @TheLacus. Would you like this one part of 0.10.26, or hold off for a little while longer?

**Comment by TheLacus** [comment]
I prefer to address the limitation with asset replacements first.

**Comment by XJDHDR** [comment]
@TheLacus 
I took a look at your latest commit and described what I found here: https://forums.dfworkshop.net/viewtopic.php?p=47637

**Comment by XJDHDR** [comment]
As I noted in [this forum post](https://forums.dfworkshop.net/viewtopic.php?p=47728#p47728), it looks like this PR is ready to go. If it can be merged for the .26 release, that would be great but it looks like it may be too late for that.

**Comment by XJDHDR** [comment]
@Interkarma

**Comment by Interkarma** [comment]
If @TheLacus is happy (and I see he's marked it ready), I'm happy to review and merge for 0.10.26 builds.

**Comment by TheLacus** [comment]
Sure :)

**Pull request merged**